app/src/main/java/org/greenstand/android/TreeTracker/orgpicker/OrgPickerScreen.kt

```diff
@@ -37,9 +41,21 @@ fun OrgPickerScreen(viewModel: OrgPickerViewModel = viewModel(factory = LocalVie
 
     Scaffold(
         topBar = {
-            ActionBar(
-                centerAction = { TopBarTitle() }
-            )
```

Comment:
I think it would be nice to keep this to keep the design constant with the dashboard and language select screens. None of them have a title on top, they use a logo.

Reply:
Since this screen is going to show different organizations available at the same time it's better we leave the header independent of an organisation.

Reply:
Alternatively we could just use Greenstand logo permanently.

Reply:
I think leaving as text makes sense in this case.